resources/views/layouts/password.blade.php

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="{{ app()->getLocale() }}">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="shortcut icon" type="image/x-icon" href="https://pbs.twimg.com/profile_images/521685439820742657/8B2oQKmP_400x400.jpeg" />
```

Comment:
What is this image `https://pbs.twimg.com/profile_images/521685439820742657/8B2oQKmP_400x400.jpeg`
ProcessMaker already have a favicon please use it.